src/Compilers/CSharp/Portable/Emitter/EditAndContinue/CSharpSymbolMatcher.cs

```diff
@@ -282,7 +282,7 @@ private sealed class MatchSymbols : CSharpSymbolVisitor<Symbol?>
         {
             private readonly IReadOnlyDictionary<AnonymousTypeKey, AnonymousTypeValue> _anonymousTypeMap;
             private readonly IReadOnlyDictionary<SynthesizedDelegateKey, SynthesizedDelegateValue> _anonymousDelegates;
-            private readonly IReadOnlyDictionary<string, AnonymousTypeValue> _anonymousDelegatesWithFixedTypes;
+            private readonly IReadOnlyDictionary<string, AnonymousTypeValue> _anonymousDelegatesWithIndexedNames;
```

Comment:
>_anonymousDelegatesWithIndexedNames [](http://example.com/codeflow?start=77&length=35)

Do we need to make any adjustments beyond the rename? Perhaps, taking default values and params into account in new scenarios, like you've done in another PR?

Reply:
While renaming, I was looking for this, that's why I created that other PR. I haven't seen any other opportunities for adjustments.